FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -2139,7 +2375,77 @@ components:
           description: The success message provided back to the client
       required:
         - message
-    
+    RecurringSchedule:
+      type: object
+      required:
+        - componentId
+        - schedules
+      properties:
+        componentId:
+          type: string
+          format: uuid
+          description: |
+            The unique identifier for the component. 
+        schedules:
+          type: array
+          items:
+            type: object
+            required:
+              - days
+              - periods
+            properties:
+              days:
+                type: array
+                items:
+                  type: string
+                  enum:
+                    - Monday
+                    - Tuesday
+                    - Wednesday
+                    - Thursday
+                    - Friday
+                    - Saturday
+                    - Sunday
+                    - All
+                description: |
+                  Specifies the days of the week that the periods apply to.
+                  - Users can list individual days (e.g., Monday, Tuesday, All) to specify scheduling data for those days.
+                  - If 'All' is selected, the component will be marked as unavailable for the entire week.
+              periods:
+                type: array
+                items:
+                  type: object
+                  description: |
+                    Defines specific periods for the schedule.
+                    - The `periods` array must be present, but individual `start` and `end` fields are optional.
+                    - If `periods` is provided as an empty array, the schedule will apply for the entire day (`T00:00:00Z` to `T23:59:59Z`).
+                    - If `start` and `end` are specified, they must be provided in ISO 8601 format. Start and end times can be in UTC (e.g., `T11:41:23Z`) or with a timezone offset (e.g., `T11:41:23-05:00`).
+                  properties:
+                    start:
+                      type: string
+                      description: |
+                        Start time of the period, in ISO 8601 format.
+                        - The format is THH:MM:SSZ (Z represents UTC) or THH:MM:SS-05:00 / THH:MM:SS+02:00 (with timezone offsets).
```

Comment:
The 'T' is optional in this context because we are limiting it to times only. Both with and without T should be acceptable here given how prescriptive we're being https://en.m.wikipedia.org/wiki/ISO_8601#Times

Reply:
Similar vein, examples should be updated to include with and without the T